chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -30,6 +30,7 @@ use actix::{
     Actor, ActorFuture, Addr, Arbiter, AsyncContext, Context, ContextFutureSpawner, Handler,
     Recipient, Running, StreamHandler, WrapFuture,
 };
+use anyhow::Context as _Context;
```

Comment:
Note: you generally want `use anyhow::Context as _;`. That CLion bugs triggers in very rare circumstances, and I belive it was fixed after we reported it